dsl/src/test/kotlin/com/chutneytesting/kotlin/dsl/ChutneyEnvironmentDslTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.chutneytesting.kotlin.dsl
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class ChutneyEnvironmentDslTest {
+
+    @Test
+    fun `Test duplicate taget name in environment`() {
+        val targets: List<ChutneyTarget> = listOf(
+            ChutneyTarget("toto", "url1"),
+            ChutneyTarget("toto","url2")
+        )
+
+        assertThrows<IllegalArgumentException> {
+            ChutneyEnvironment("default value", "description", targets)
+        }
```

Comment:
Assert that the error message contains both 'toto' & 'bou' as non unique targets and not 'unique'

Reply:
Fixed